data/json/items/melee/bludgeons.json

```diff
@@ -870,16 +901,17 @@
     "id": "sharp_toothbrush",
     "symbol": ",",
     "color": "light_blue",
-    "name": { "str": "sharpened toothbrush", "str_pl": "sharpened toothbrushes" },
-    "description": "A personal hygiene item turned into a shank.",
+    "name": "plastic shank",
+    "name_plural": "plastic shanks",
```

Comment:
This should be using a `name` object as it did before.

Reply:
updated!